screens/Home.js

```diff
@@ -19,29 +19,134 @@ import {
   Menu,
   MenuOptions,
   MenuOption,
-  MenuTrigger, renderers
+  MenuTrigger,
+  renderers
 } from "react-native-popup-menu";
 import getStyleSheet from "../styles/themestyles";
 import { ScreenStyles, FontStyles } from "../styles/global";
 import firebase from "react-native-firebase";
 import DatabaseManager from "../components/DatabaseManager";
 import { showMessage } from "react-native-flash-message";
+import AppIntroSlider from "react-native-app-intro-slider";
+
+const tutorialSlides = [
+  {
+    key: "slide1",
+    title: "Welcome to FitnessGo!",
+    text:
+      "This app allows you to create timers for your workouts, discover workouts from other people, and share your own workouts with your friends.",
+    image: require("../assets/images/main/Stopwatch.png")
+  },
+  {
+    key: "slide2",
+    title: "Create workouts",
+    text: "Press plus button to create a workout",
+    image: require("../assets/images/tutorial/1.png")
+  },
+  {
+    key: "slide3",
+    title: "Add your exercises",
+    text:
+      'Type in workout name, select a workout category, and add exercise by selecting an exercise name and typing number of repetitions, duration and break. When done, simply press "Save" button on top right corner.',
+    image: require("../assets/images/tutorial/2.png")
+  },
+  {
+    key: "slide4",
+    title: "User library",
+    text:
+      "You'll see a newly created on your Home screen. All your workouts will be displayed here.",
+    image: require("../assets/images/tutorial/3.png")
+  },
+  {
+    key: "slide5",
+    title: "Workout details",
+    text:
+      'You can see workout details by simply clicking on the workout on Home screen. If you want to edit your workout, press "Edit" button on top right corner.',
```

Comment:
You can see workout details by simply clicking on the workout on the "Home" screen. If you want to edit your workout, press the "Edit" button in the top right corner.